clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/security/sdkclient/AmazonClientInvocationHandler.java

```diff
@@ -274,6 +274,16 @@ public DescribeLoadBalancersResult describeLoadBalancers(DescribeLoadBalancersRe
         describe(request, "loadBalancerNames", "loadBalancers", LoadBalancerDescription.class));
   }
 
+  public com.amazonaws.services.elasticloadbalancingv2.model.DescribeLoadBalancersResult describeLoadBalancersV2() {
```

Comment:
I don't think this is necessary - this class is a proxy handler so it looks for the same method signature on the underlying SDK class (I don't think the elbv2 api call is describeLoadBalancersV2?)